triples-generator/src/test/scala/ch/datascience/triplesgenerator/reprovisioning/IOTriplesRemoverSpec.scala

```diff
@@ -43,7 +54,7 @@ class IOTriplesRemoverSpec extends AnyWordSpec with InMemoryRdfStore with should
         .removeAllTriples()
         .unsafeRunSync() shouldBe ((): Unit)
 
-      rdfStoreSize shouldBe 0
+      rdfStoreSize shouldBe cliVersionJsonLD.properties.size + 1
```

Comment:
I believe the `+ 1` stands for the `rdf:type` triple. Maybe a bit of a comment would be nice here?